tequilapi/endpoints/node.go

```diff
@@ -120,7 +121,49 @@ func (ne *NodeEndpoint) GetProviderSessions(c *gin.Context) {
 		return
 	}
 
-	utils.WriteAsJSON(contract.ProviderSessionsResponse{Statuses: res}, c.Writer)
+	utils.WriteAsJSON(contract.NewProviderSessionsResponse(res), c.Writer)
+}
+
+// GetProviderSessionsCount A number of sessions during a period of time
+// swagger:operation GET /node/provider/sessions-count provider GetProviderSessionsCount
+// ---
+// summary: Provides Node sessions number during a period of time
+// description: Node sessions count during a period of time
+// parameters:
+//   - in: query
+//     name: range
+//     description: period of time ("1d", "7d", "30d")
```

Comment:
@soffokl would `90d` range be performance problem?

Reply:
yep, it will be too slow for big providers. I just tested and it failed to finish in 30 seconds.